aspnetcore/tutorials/web-api-help-pages-using-swagger.md

```diff
@@ -110,239 +80,217 @@ In Visual Studio, press ^F5 to launch the app and navigate to `http://localhost:
+                },
+                "isComplete": {
+                    "default": false,
+                    "type": "boolean"
+                }
+            }
        }
    },
    "securityDefinitions": {}
-   }
-   ```
+}
+```
 
-This document is used to drive the Swagger UI which can be viewed by navigating to `http://localhost:<random_port>/swagger`
+This document drives the Swagger UI, which can be viewed by navigating to `http://localhost:<random_port>/swagger`:
 
 ![Swagger UI](web-api-help-pages-using-swagger/_static/swagger-ui.png)
 
-Each of the methods in the ToDo controller can be tested from the UI. Tap a method to expand the section, add any necessary parameters and tap "Try it out!".
+Each public method in `TodoController` can be tested from the UI. Tap a method name to expand the section. Add any necessary parameters, and tap "Try it out!".
 
 ![Example Swagger GET test](web-api-help-pages-using-swagger/_static/get-try-it-out.png)
 
 ## Customization & Extensibility
 
-Swagger is not only a simple way to represent the API, but has options for documenting the object model, as well as customizing the interactive UI to match your look and feel or design language.
+Swagger provides options for documenting the object model and customizing the UI to match your theme.
 
 ### API Info and Description
 
-The config. action passed to the `AddSwaggerGen` method can be used to add information such as the author, license, description.
+The configuration action passed to the `AddSwaggerGen` method can be used to add information such as the author, license, and description:
 
-```csharp
-services.AddSwaggerGen(c =>
-{
-	c.SwaggerDoc("v1", new Info
-	{
-		Version = "v1",
-		Title = "ToDo API",
-		Description = "A simple example ASP.NET Core Web API",
-		TermsOfService = "None",
-		Contact = new Contact { Name = "Shayne Boyer", Email = "", Url = "https://twitter.com/spboyer"},
-		License = new License { Name = "Use under LICX", Url = "https://example.com/license" }
-	});
-});
-```
+[!code-csharp[Main](../tutorials/web-api-help-pages-using-swagger/sample/TodoApi/Startup.cs?range=20-30,36)]
 
-The following image shows the Swagger UI displaying the version information added.
+The following image depicts the Swagger UI displaying the version information:
 
 ![Swagger UI with version information: description, author, and see more link](web-api-help-pages-using-swagger/_static/custom-info.png)
 
 ### XML Comments
 
-To enable XML comments, right click the project in Visual Studio and select **Properties** and then check the **XML Documentation file** box under the **Output Settings** section.
+To enable XML comments, right-click the project in Visual Studio and select **Properties**. Check the **XML Documentation file** box under the **Output Settings** section.
 
 ![Build tab of project properties](web-api-help-pages-using-swagger/_static/swagger-xml-comments.png)
 
 Configure Swagger to use the generated XML file.
 
 > [!NOTE]
```

Comment:
Do we really need this note? If so consider moving it someplace else and make it a text note.